### ISSUE TITLE
MEDIA_ERR_SRC_NOT_SUPPORTED when updating source

### ISSUE THREAD

**Issue by Palethorn** [videojs/videojs-contrib-dash#73]
Hello, I have an issue when updating source on initialized player object. Sample code
```javascript
            window.player.src({
                type : type,
                src: val.src,
                keySystemOptions: [
                    {
                        name: 'com.widevine.alpha',
                        options: {
                            licenseUrl: widevine_la_url
                        }
                    },
                    {
                        name: 'com.microsoft.playready',
                        options: {
                            licenseUrl: playready_la_url
                        }
                    }
                ]
            });
```

This results in the following error:
> VIDEOJS: ERROR: (CODE:4 MEDIA_ERR_SRC_NOT_SUPPORTED) The media could not be loaded, either because the server or network failed or because the format is not supported

Plugin version 2.1.0

**Comment by halibegic** [comment]
+1

**Comment by forbesjo** [comment]
This error should be fixed with 2.2.0 (released yesterday). I believe the issue stems from this plugin manually clearing the video element source between src() calls which is no longer needed with dash.js 2.0.

**Comment by Palethorn** [comment]
I have updated dash.js to v2.1.1 and videojs-dash to 2.2.0. I'm having the same issue. And there is another issue as well. Video plays for a while then stops. Chunks stop downloading. Stream is live, and I'm fetching it from Wowza version 4.4.1.

**Comment by forbesjo** [comment]
Do you have this issue when running the stream through the dash.js reference player? http://dashif.org/reference/players/javascript/v2.1.1/samples/dash-if-reference-player/index.html

**Comment by Palethorn** [comment]
It works with dash.js reference player, but it could mean that it works on older version. Also, I'm testing on the latest stable version of Chrome.

**Comment by forbesjo** [comment]
Would you be able to post a sample player for debugging?

**Comment by Palethorn** [comment]
I'll try to configure public example.

**Comment by Palethorn** [comment]
I've set up a public example with debug versions of videojs-dash, dash.js and video.js here http://cavar.pw/videojs/

**Comment by forbesjo** [comment]
I believe https://github.com/videojs/videojs-contrib-dash/pull/79 will fix this issue

**Comment by forbesjo** [comment]
Now released as version 2.3.0

**Comment by Palethorn** [comment]
Great, thank you. There is still an issue of not refreshing live source, but I'll create an another issue. Closing this one down.
